get-new-tool-versions/get-new-tool-versions.ps1

```diff
@@ -0,0 +1,66 @@
+<#
+.SYNOPSIS
+Check and return list of new available tool versions
```

Comment:
Maybe we should add descriptions to .PARAMETERs?

Reply:
Thanks!